include/s3select_functions.h

```diff
@@ -86,10 +142,26 @@ class s3select_functions
     {"int", s3select_func_En_t::TO_INT},
     {"float", s3select_func_En_t::TO_FLOAT},
     {"substring", s3select_func_En_t::SUBSTR},
-    {"timestamp", s3select_func_En_t::TO_TIMESTAMP},
-    {"extract", s3select_func_En_t::EXTRACT},
-    {"dateadd", s3select_func_En_t::DATE_ADD},
-    {"datediff", s3select_func_En_t::DATE_DIFF},
+    {"to_timestamp", s3select_func_En_t::TO_TIMESTAMP},
+    {"extract_year", s3select_func_En_t::EXTRACT_YEAR},
```

Comment:
it needs to "hide" all datetime functions, since the datetime syntax rules does the association,
direct call to extract_year( args ) should be avoided

"extract_year" ==> "#extract_year#"